### PR TITLE
Cartesian speed limitation in PJTC [Doc update]

### PR DESCRIPTION
**Pull request by martiniil**

### CODE REVIEW

**Review by HiddenBug — DISMISSED** [review]
The architecture looks good to me. Maybe we should write down, that the controller itself can now trigger a stop 1.

**Comment by hslusarek** [comment]
@martiniil PR #308 tries to integrate the velocity check into the `JointTrajectoryController`.

File: prbt_hardware_support/README.md [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
+be called directly by the user. They already are included in our top-level
+launch file (for more information on how-to run the PRBT robot with ROS see the 
+[pilz_robots-README](https://github.com/PilzDE/pilz_robots#running-on-the-real-robot).
+
+# Supported DIN EN ISO 10218-1 safety features
+The prbt_hardware_support package provides support for a number of safety
+features which are essential for a DIN EN ISO 10218-1 certifiable robot system.
+  
+**Please note:**  
+The DIN EN ISO 10218-1 support is WORK IN PROGRESS!!!
```
Comment (ct2034) [inline prbt_hardware_support/README.md:39]:
```suggestion
The DIN EN ISO 10218-1 support is currently **WORK IN PROGRESS**  
```

File: prbt_hardware_support/README.md [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
+**Please note:**  
+The DIN EN ISO 10218-1 support is WORK IN PROGRESS!!!  
+ 
+## Needed/supported hardware
+In order for the safety features to work, one needs special hardware
+components supporting special safety feature.
+Currently, we test all our features against the following 
+hardware setup:
+- Robot: Manipulator module PRBT
+- Safety controller: PSS 4000 (with special PSS 4000 program)
```
Comment (ct2034) [inline prbt_hardware_support/README.md:47]:
```suggestion
- Safety controller: PSS 4000 (with a dedicated program)
```

File: prbt_hardware_support/README.md [hunk 3]
```diff
@@ -1,21 +1,75 @@
...
+- Operating mode selector switch: PITmode
+- Enabling switch: PITenable
+- Pushbutton unit (with emergency stop): PITgatebox
```
Comment (ct2034) [inline prbt_hardware_support/README.md:50]: this can have various configurations

File: prbt_hardware_support/README.md [hunk 4]
```diff
@@ -1,21 +1,75 @@
...
+
+**Please note:**  
+Not all safety features in this section are supported by both PILZ safety 
+controllers! Currently, the PNOZmulti does not support the brake test feature
```
Comment (ct2034) [inline prbt_hardware_support/README.md:54]: this is the first time, PNOZmulti is mentioned. Either introduce it as (legacy, unsupported) option or leave it out altogether.

File: prbt_hardware_support/README.md [hunk 5]
```diff
@@ -1,21 +1,75 @@
...
+In the [system overview section](#overview-system-components),
+you can find a component diagram showing the overall architecture of our
+system. The component diagram shows all our nodes and the connections 
+between them.
+
+**Please note:**  
+The launch files included in the `prbt_hardware_support` package don't need to
+be called directly by the user. They already are included in our top-level
+launch file (for more information on how-to run the PRBT robot with ROS see the 
+[pilz_robots-README](https://github.com/PilzDE/pilz_robots#running-on-the-real-robot).
```
Comment (ct2034) [inline prbt_hardware_support/README.md:32]: we could reference https://github.com/PilzDE/pilz_templates/ here

File: prbt_hardware_support/README.md [hunk 6]
```diff
@@ -1,21 +1,75 @@
...
+# Supported DIN EN ISO 10218-1 safety features
+The prbt_hardware_support package provides support for a number of safety
+features which are essential for a DIN EN ISO 10218-1 certifiable robot system.
+  
+**Please note:**  
+The DIN EN ISO 10218-1 support is WORK IN PROGRESS!!!  
+ 
+## Needed/supported hardware
+In order for the safety features to work, one needs special hardware
+components supporting special safety feature.
```
Comment (ct2034) [inline prbt_hardware_support/README.md:43]:
```suggestion
components supporting the required safety features.
```

File: prbt_hardware_support/README.md [hunk 7]
```diff
@@ -1,21 +1,75 @@
...
+Currently, we test all our features against the following 
+hardware setup:
+- Robot: Manipulator module PRBT
```
Comment (ct2034) [inline prbt_hardware_support/README.md:46]:
```suggestion
- Robot: Manipulator module PRBT 6
```

File: prbt_hardware_support/README.md [hunk 8]
```diff
@@ -1,21 +1,75 @@
...
+  - [Needed/supported hardware](#neededsupported-hardware) 
+  - [Definitions](#definitions)
+  - [Safe stop 1](#safe-stop-1-ss1)
+  - [Brake test](#brake-test)
+  - [Operation modes](#operation-modes)
+  - [Speed monitoring](#speed-monitoring)
+- [Overview system components](#overview-system-components)
+
+# Introduction
+The `prbt_hardware_support` package provides support for the Pilz hardware
```
Comment (ct2034) [inline prbt_hardware_support/README.md:13]: There is a video on this: https://vimeo.com/378682930 ;-)

File: prbt_hardware_support/README.md [hunk 9]
```diff
@@ -39,17 +93,33 @@ These modes are:
...
 See DIN EN ISO 10218-1 for more details or contact us: ros@pilz.de
 
-## Note
-In operation mode T1 the robot can be moved as usual. However, if an attempt to exceed the speed limit of 250 mm/s in T1 is detected, the prevailing motion is aborted and a controlled stop is performed.
-
-# Architecture
+**Please note:**  
+In operation mode T1 the robot can be moved as usual. 
+However, if an attempt to exceed the speed limit of 250 mm/s in T1 is detected,
+the prevailing motion is aborted and a controlled stop is performed.
```
Comment (ct2034) [inline prbt_hardware_support/README.md:99]:
```suggestion
the current motion is aborted and a controlled stop is performed.
```

**Review by ct2034 — APPROVED** [review]
review of all, except the architecture diagram

**Review by ct2034 — CHANGES_REQUESTED** [review]
sorry, I approved by accident. Please see my previous comments

**Comment by martiniil** [comment]
Merge via #308.

**Pull request closed without merging**